pm/sender.go

```diff
@@ -113,7 +114,7 @@ func (s *sender) CreateTicketBatch(sessionID string, size int) (*TicketBatch, er
 	// Ensure backwards compatbility
 	// If no expirationParams are included by O
 	// B sets the values based upon its last seen round
-	if expirationParams == nil {
+	if expirationParams == nil || (expirationParams.CreationRound == 0 && expirationParams.CreationRoundBlockHash == ethcommon.Hash{}) {
```

Comment:
```golang
expirationParams == nil || expirationParams.CreationRound == 0 || expirationParams.CreationRoundBlockHash == ethcommon.Hash{}
```

Should the conditional be the above instead? If either field in `expirationParams` is the zero value the sender would set the expiration params on its own.

Reply:
Yes although both values are always used in tandem so it's never either.

We _could_ even skip the `expirationParams == nil` check since it should never be nil (due to the usage of `pmticketparams`

Reply:
fixed in c3bcc39f